### DISCUSSION TITLE
Failed to load webview resources

### DISCUSSION THREAD

**Discussion by Jay-Bhatt98** [eclipse-theia/theia#13056] [General]
<img width="1776" alt="Screenshot 2023-11-06 at 2 13 05 PM" src="https://github.com/eclipse-theia/theia/assets/104554325/9504bdd1-43b8-41cb-ab51-5a5e91b24053">
i am using thunder vsix extention with theia 1.40.0 version but i am getting this error when i try to call any api and same type of error occuring in other extention also bundle.js/main.js/min.css not found

**Comment by JonasHelming** [comment]
Could you try this with the latest Theia release and if it still occurs open a bug report?
